package.json

```diff
@@ -43,13 +45,15 @@
   "bin": {
     "blockstack-subdomain-registrar": "lib/index.js"
   },
+  "prettier": "@stacks/prettier-config",
   "scripts": {
     "start": "npm run build && node lib/index.js",
     "start:regtest": "BSK_SUBDOMAIN_CONFIG=config-develop.json BSK_SUBDOMAIN_REGTEST=true npm run start",
     "build": "babel src -d lib",
     "flow": "flow",
-    "lint": "eslint src",
-    "lint:fix": "eslint src --fix",
+    "lint": "eslint src &&  npm run lint:prettier",
```

Comment:
maybe let's move eslint to a separate script `lint:eslint` https://github.com/hirosystems/stacks-blockchain-api/blob/master/package.json#L30?